community/psad/APKBUILD

```diff
@@ -35,16 +36,19 @@ build() {
 		-e "s|/usr/sbin/fwcheck_psad|$pkgdir&|" \
 		-e "s|/usr/sbin/kmsgsd|$pkgdir&|" \
 		-e "s|/usr/sbin/psad|$pkgdir&|" \
-		-i ./psad.conf || return 1
+		-i ./psad.conf
 
 	# Set mail command to mailx.
-	sed -e 's|/bin/mail;|/usr/bin/mail;|g' -i ./psad.conf || return 1
+	sed -e 's|/bin/mail;|/usr/bin/mail;|g' -i ./psad.conf
 
 	# Disable install of generic init script & setting numeric run level.
 	local start=$(sed -n '/if ($init_dir and &is_root()) {/=' ./install.pl)
-	local end=$(expr $start + 7)
-	# Busybox sed does not support +7d.
-	sed ''$start','$end'd' ./install.pl -i || return 1
+	local end=$(expr $start + 11)
+	# Busybox sed does not support +11d.
+	sed ''$start','$end'd' ./install.pl -i
+
+	# fix for building on Travis
+	sed -i "s|^my \$is_systemd =.*|my \$is_systemd = 0;|" ./install.pl
```

Comment:
I generallly prefer use patches instead of sed.

Reply:
resolved in https://github.com/alpinelinux/aports/pull/5076